src/modules/launcher/PowerLauncher/MainWindow.xaml

```diff
@@ -52,6 +52,7 @@
         <local:ResultList
             x:Name="ListBox"
             Grid.Row="1"
+            Margin="12,0,0,0"

```

Comment:
Please make this symmetrical

Reply:
When showing multiple results the scrollbar provides the correct spacing. I think this is something we should fix in `WpfUi` as the behavior is not correct

![image](https://github.com/microsoft/PowerToys/assets/9866362/39a62686-3a37-4238-b904-0eccc1fc2bc1)

Reply:
#30318 tracking issue, will look at what we can fix in `WpfUI` and update to a new preview in a seperate PR!

Reply:
@Jay-o-way FYI, fix in PR: https://github.com/lepoco/wpfui/pull/873